docs/how-to/h-deploy-aks.md

```diff
@@ -0,0 +1,263 @@
+```
+[/details]
+
+## Deploy Charms
+
+Create a new Juju model, if needed:
+```shell
+juju add-model <MODEL_NAME>
+```
+
+[note type="caution"]
+(Optional) Increase the debug level if you are troubleshooting charms:
+```shell
+juju model-config logging-config='<root>=INFO;unit=DEBUG'
+```
+[/note]
+
+Then, Charmed Kafka can be deployed as usual:
+
+```shell
+juju deploy zookeeper-k8s -n3 --channel 3/stable
+juju deploy kafka-k8s -n3 --channel 3/stable
+juju integrate kafka-k8s zookeeper-k8s
+```
+
+We also recommend to deploy a [Data Integrator](https://charmhub.io/data-integrator) for creating an admin user to manage the content of the Kafka cluster:
+
+```shell
+juju deploy data-integrator admin --channel edge \
+  --config extra-user-roles=admin \
+  --config topic-name=admin-topic
+```
+
+And integrate it with the Kafka application:
+
+```shell
+juju integrate kafka-k8s admin
+```
+
+For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](/t/charmed-kafka-how-to-manage-app/10285) user guide.
+
+## Display deployment information
+
+Display information about the current deployments with the following commands:
+
+```shell
+kubectl cluster-info 
+```
+
+[details="Sample output:"]
+```shell
+Kubernetes control plane is running at https://aks-user-aks-aaaaa-bbbbb.hcp.eastus.azmk8s.io:443
+CoreDNS is running at https://aks-user-aks-aaaaa-bbbbb.hcp.eastus.azmk8s.io:443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://aks-user-aks-aaaaa-bbbbb.hcp.eastus.azmk8s.io:443/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+```
+[/details]
+
+```
+az aks list
+```
+[details="Sample output:"]
+```shell
+...
+        "count": 1,
+        "currentOrchestratorVersion": "1.28.9",
+        "enableAutoScaling": false,
+...
+```
+[/details]
+
+```
+kubectl get node
+```
+
+[details="Sample output:"]
+```shell
+NAME                                STATUS   ROLES   AGE   VERSION
+aks-nodepool1-31246187-vmss000000   Ready    agent   11m   v1.28.9
+```
+[/details]
+
+## Clean up 
+
+[note type="caution"]
+Always clean AKS resources that are no longer necessary -  they could be costly!
+[/note]
+
+To clean the AKS cluster, resources and juju cloud, run the following commands:
+
+```shell
+juju destroy-controller <CONTROLLER_NAME> --destroy-all-models --destroy-storage --force
+```
+List all services and then delete those that have an associated EXTERNAL-IP value (load balancers, ...):
+```shell
+kubectl get svc --all-namespaces
+kubectl delete svc <SERVICE_NAME> 
+```
+Next, delete the AKS resources (source: [Deleting an all Azure VMs]((https://learn.microsoft.com/en-us/cli/azure/delete-azure-resources-at-scale#delete-all-azure-resources-of-a-type) )) 
+```shell
+az aks delete -g <RESOURCE_GROUP> -n <K8S_CLUSTER_NAME>
+```
+Finally, logout from AKS to clean the local credentials (to avoid forgetting and getting exposed to a risk of leaking credentials):
```

Comment:
```suggestion

Finally, logout from AKS to clean the local credentials (to avoid forgetting and getting exposed to a risk of leaking credentials):

```